ui/page/exchange/create_order_page.go

```diff
@@ -255,7 +257,7 @@ func (pg *CreateOrderPage) displayCreateWalletModal(isSourceWallet bool, asset l
 		SetCancelable(true).
 		SetContentAlignment(layout.Center, layout.W, layout.Center).
 		SetPositiveButtonCallback(func(_ bool, _ *modal.InfoModal) bool {
-			pg.ParentNavigator().Display(components.NewCreateWallet(pg.Load, func() {
+			pg.ParentWindow().Display(components.NewCreateWallet(pg.Load, func() {
```

Comment:
Errm, #389. I think it's consistent to show the form on the same master page as the create order page